csrankings-f.csv

```diff
@@ -335,6 +335,7 @@ Filip Rozpedek,University of Massachusetts Amherst,https://sites.google.com/view
 Filip Sadlo,Heidelberg University,https://vcg.iwr.uni-heidelberg.de/people/sadlo,aptHqwMAAAAJ
 Filip Sieczkowski,University of Wroclaw,https://sites.google.com/a/cs.uni.wroc.pl/efes,aNLxzEkAAAAJ
 Filip Zelezný,Czech Technical University,http://ida.felk.cvut.cz/zelezny,gt3emoEAAAAJ
+Filipa Correia,Universidade de Lisboa,https://filipacorreia.com/,lEjl_5MAAAAJ

```

Comment:
Listed as "assistant researcher", not obviously TT faculty. Please clarify.

Reply:
This is not an obvious scenario. Filipa was hired under a new scheme from the Portuguese government (https://www.fct.pt/en/concursos/fct-tenure-1-edicao) in which public institutions can hire PhDs for tenure-track positions that do not entail any teaching. So, while she is on a tenure-track position, I'm not sure if this qualifies as "faculty".

Reply:
Got it. Please remove.

Reply:
Not removed, should not have been resolved.